test/csuite/timestamp_abort/main.c

```diff
@@ -374,9 +640,14 @@ thread_ckpt_run(void *arg)
     (void)unlink(ckpt_file);
     testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
     first_ckpt = true;
+
+    /*
+     * Create checkpoints until we get killed.
+     */
     for (i = 1;; ++i) {
         sleep_time = __wt_random(&td->extra_rnd) % MAX_CKPT_INVL;
         testutil_tiered_sleep(opts, session, sleep_time, &flush_tier);
+
```

Comment:
Both of these changes to `thread_ckpt_run` no longer seem necessary since you moved the backup code out.